### DISCUSSION TITLE
Project helper for  Tri and Tetra regions

### DISCUSSION THREAD

**Discussion by ZAARAOUI999** [adtzlr/felupe#299] [Q&A] [answered]
Hello Mr Andreas, is there any way to let the project helper work with RegionTriangle and RegionTetra as it does with RegionQuad and RegionHexahedron?

**Comment by adtzlr** [comment, marked as answer]
Hi, 

actually, I have never thought on how to realize that. For quads and hexahedrons, the so-called full-integrated element formulations have equal number of mesh-points and integration points. For triangles and tetrahedrons, this is not the case.

For linear triangles and tetrahedrons, the field gradients are constant and you could output cell-values like a stress tensor directly. Even for quadratic elements, this is a super-simple approach (evaluate cell-based *mean* values).

**Reply by ZAARAOUI999** [reply]
okey, that's clear, Thank you.
